divi/src/CoinMinter.cpp

```diff
@@ -12,10 +12,15 @@
 #include <boost/thread.hpp>
 #include <SuperblockHelpers.h>
 #include <BlockFactory.h>
-#include <main.h>
 #include <ValidationState.h>
+#include <txmempool.h>
 
 extern const int nHashDrift;
+extern int64_t nReserveBalance;
+extern CChain chainActive;
```

Comment:
Remove the chainActive extern, and notice that the member activeChain_ object should have the call made on it instead

Reply:
done